docs/rest_api.md

```diff
@@ -492,6 +590,14 @@ separation. Additionally, headnodes may have an arbitrary number of
 nics, and so being able to attach two networks to the same nic is not as
 important.
 
+Authorization requirements:
+
+* Access to the project which owns `<headnode>`.
+* If the network is not public, access to the project named by it's
```

Comment:
it's=its

Also, similar comment to above re: moving note to `Possible errors`.